feature/platform/transceiver/zr_db_q_value_test/README.md

```diff
@@ -0,0 +1,62 @@
+# TRANSCEIVER-6: Telemetry: 400ZR Optics Q-value streaming.
+
+## Summary
+
+Validate 400ZR optics module reports Q-value performance data as defined in
+module CMIS VDM(Versatile Diagnostics Monitor).
```

Comment:
Is the term VDM vendor agnostic?

Reply:
Thanks for pointing out, for some reason I imagined it being a vendor specific acronym :)

Reply:
You are right this is defined in the CMIS :)